tests/src/helpers.c

```diff
@@ -31,7 +31,17 @@ mbedtls_threading_mutex_t mbedtls_test_info_mutex;
 #endif /* MBEDTLS_THREADING_C */
 
 /*----------------------------------------------------------------------------*/
-/* Mbedtls Test Info accessors */
+/* Mbedtls Test Info accessors
+ *
+ * NOTE - there are two types of accessors here; the _internal functions, which
+ * are expected to be used from this module *only*. These do not attempt to lock
+ * the mbedtls_test_info_mutex, as they are designed to be called from within
+ * public functions which do lock the mutex first (if mutexes are enabled).
+ * The main reason for this is the need to set some sets of test data
```

Comment:
`set some sets` sounds awkward - is there another way to phrase that without the repetition? (I'm looking at a thesaurus right now!)

Reply:
Something like `need to set multiple test data values atomically`?